config/sync/block.block.views_block__right_sidebar_latest_revision_block_1.yml

```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: "/node/*/edit\r\n/node/*/history\r\n/node/*/delete\r\n/node/*/clone\r\n/node/*/moderation-history\r\n"
+    pages: "/node/*/edit\r\n/node/*/history\r\n/node/*/delete\r\n/node/*/clone\r\n/node/*/moderation-history\r\n/node/*/translations\r\n"
```

Comment:
This hides the right sidebar view from the node/nid/translations screen (much like it's hidden from the node/nid/edit screen).

Reply:
Great idea to make it more similar.